src/components/shared/AboutDialog/AboutDialog.tsx

```diff
@@ -30,6 +31,10 @@ const AboutDialog: FC<AboutDialogProps> = ({open, onClose}) => {
                 <Button startIcon={<GitHubIcon/>} color="inherit" variant="outlined" onClick={() => handleExternalClick("https://github.com/nthnluu/hours-frontend/wiki/Contributing")}>
                    Contribute
                 </Button>
+                <Divider/>
+                <Typography>
+                    In loving memory of Will Povell '20. We'll miss you.
```

Comment:
nit: "We miss you"?

Reply:
this is the original; feel free to change it